src/components/Table/TableRowComponent.jsx

```diff
@@ -49,9 +52,25 @@ class TableRow extends Component {
       const data = rowItem[column];
 
       if (typeof data === 'object') {
+        const popoverContent = <CodeMirror value={jsyaml.safeDump(data)}
+          options={{
+            mode: 'yaml',
+            lineNumbers: true,
+            readOnly: true,
+            cursorBlinkRate: -1
+          }}
+        />;
+        const popover = data === null ? 'null' : <Popover content={popoverContent}
+          interactionKind={PopoverInteractionKind.CLICK}
+          popoverClassName="pt-popover-content-sizing"
+          position={Position.RIGHT}
+          useSmartPositioning={true}>
+          <button className="pt-button pt-intent-primary">Show</button>
+        </Popover>;
+
```

Comment:
I think that there should be `if` construction rather than ternary operator `?:`